### ISSUE TITLE
Feature Request - Make CRTLD a selectable option in the settings

### ISSUE THREAD

**Issue by wellloaded** [Windscribe/Desktop-App#136]
Personally speaking I don't use CTRLD however just because Windscribe is installed I get my NIC DNS forcefully set to 127.0.0.1.
Now... on this particular device I only use Windscribe so it seems like my DNS pointing to 127.0.0.1 prevents me from connecting (resolving names) unless WIndscribe (and CTRLD with it) is running/open.

You need to think of users like me that occasionally travel and can't in certain circumstances use Windscribe in certain sites. Currently the DNS defaulting to 127.0.0.1 is an issue for us. I would strongly suggest you add an option to clearly and centrally control CTRLD. It makes no sense to me to forcely have CTRLD running to then tell it to use my router IP as a resolver (where I'm using Stubby + DNSSEC).

So bottom line, please allow users to enable/disable CTRLD and especially restore the DNS set by DHCP when this option is disabled.

Thanks

**Comment by bernerdad** [comment]
CTRLD is only used if you set the `Connected DNS` to Custom.  You can disable it off by selecting `Auto' instead.

**Comment by bernerdad** [comment]
`Auto` will use R.O.B.E.R.T. as described [here](https://windscribe.com/features/flexible-dns).  We will be implementing an improvement similar to how the old `Custom` setting worked.   It will allow you to set the custom DNS IP to a LAN IP and that will disable the use of ctrld.  This will likely get implemented in the 2.11 release.

**Comment by wellloaded** [comment]
Amazing, thank you for listening to the users :-)

**Comment by wellloaded** [comment]
Sorry I need to reopen this as there's still something strange even in the current stable release.

Even if I set the Windscribe Windows client to DNS resolve to Auto, my DNS for the WIFi card still defaults to 127.0.0.1 so it's not set by DHCP as expected.

![image](https://github.com/Windscribe/Desktop-App/assets/46318040/53446922-4e40-4986-b08a-ac8c56059130)

![image](https://github.com/Windscribe/Desktop-App/assets/46318040/a39d7d72-9e37-4eb9-a631-6fcdb58111d6)

![image](https://github.com/Windscribe/Desktop-App/assets/46318040/9bad114b-07d9-4d12-9ade-8fbf9cbf248d)

The only way I can make it move away from 127.0.0.1 is to e it manually but you need to understand this is not practical at all as moving to different networks I need to change the DNS IP manually every time.

**Comment by bernerdad** [comment]
I'm a bit puzzled by this one.  Are you saying your Wi-Fi adapter's DNS server address is correct when the Windscribe app is not running, and gets changed to 127.0.0.1 when you launch the Windscribe app?

**Comment by wellloaded** [comment]
Ok let me try to rephrase the issue also adding more details:

- DNS Windscribe settings: Auto
- Whether Windscribe is connected or disconnected my WIFI NIC adapter's DNS always points to 127.0.0.1 despite this being set to DHCP, and this is unwanted.
- This issue happens even if the Windscribe app is not open.

Now let me be completely honest with you, I have no proof this to be caused by Windscribe specifically but admittedly this is the only app from my list of installed apps that is known to be fiddling with DNS settings at lowe level (without user interaction). Also a **netstat -a -b** confirms there's no service listening on 127.0.0.1:53 regardless (unless I run CTRLD of course).

I have done a network reset (build in WIndows 11 feature) but I'm still experiencing the issue.

I must sound desperate :-)

**Comment by bernerdad** [comment]
Your best bet here to rule out/in the Windscribe app is to:
- Uninstall the Windscribe app.
- Reboot (a Windows fan favorite).
- Get Win11 to reset your NIC settings to their defaults.
- See if the issue still occurs now that the Windscribe app is gone.

**Comment by wellloaded** [comment]
The solution to this latest issue was to remove the WiFi device and delete the drivers. I then reinstalled the latest drivers form Intel and not it works. Re-closing this.

**Comment by bernerdad** [comment]
Just a heads up that the new v2.10.6 release implements the above noted feature, allowing you to set the custom DNS IP to a IPv4 IP and that will disable the use of ctrld.
